client-core/src/cipher/default.rs

```diff
@@ -169,6 +169,9 @@ impl TransactionObfuscation for DefaultTransactionObfuscation {
                     tx, state, witness,
                 )))
             }
+            SignedTransaction::UnjailTransaction(_, _) => {
+                unreachable!("unjailing does not need enclave")
```

Comment:
same for unbondtx -- currently, it's being returned as `return Ok(TxAux::UnbondStakeTx(tx, witness));` -- @devashishdxt may comment what's preferred there (where unreachable panic or identity function?)

Reply:
because SignedTransaction::UnbondStakeTransaction, SignedTransaction::UnjailTransaction are not used, i'll remove from enum.